lang/LangPrimSource/PyrArrayPrimitives.cpp

```diff
@@ -2087,6 +2089,96 @@ int prArrayContainsSeqColl(struct VMGlobals *g, int numArgsPushed)
 	return errNone;
 }
 
+inline int sc_arrayMaxDepth(PyrSlot *a, int depth)
```

Comment:
Hair-splitting C++ question — is it a good idea to inline a large recursive function? It doesn't seem like this function is getting used frequently enough to warrant an `inline` considering the extra space it would potentially take up in the executable.